pkg/validation/component.go

```diff
@@ -60,27 +60,33 @@ type Component interface {
 
 // Component is the management component.
 type component struct {
-	keycloakClient      KeycloakClient
-	tokenProvider       TokenProvider
-	archiveDBModule     ArchiveDBModule
-	eventReporterModule EventsReporterModule
-	accredsService      AccreditationsServiceClient
-	configDBModule      ConfigurationDBModule
-	logger              log.Logger
-	originEvent         string
+	keycloakClient       KeycloakClient
+	tokenProvider        TokenProvider
+	archiveDBModule      ArchiveDBModule
+	eventReporterModule  EventsReporterModule
+	accredsService       AccreditationsServiceClient
+	configDBModule       ConfigurationDBModule
+	logger               log.Logger
+	originEvent          string
+	unknownAgentRealm    string
```

Comment:
Why not to use constants instead of field of a struct ?

Reply:
I just copied what was done for unknownAgentUserID and unknownAgentUsername... You ask yourself why you did that for those 2 fields! ;-)